e2e/imgbuild/imgbuild.go

```diff
@@ -33,74 +33,61 @@ func (c imgBuildTests) buildFrom(t *testing.T) {
 	// see https://github.com/sylabs/singularity/issues/4407
 	tt := []struct {
 		name       string
-		dest       string
 		dependency string
 		buildSpec  string
-		sandbox    bool
 	}{
 		{
 			name:      "BusyBox",
-			dest:      c.env.TestDir + "/container",
 			buildSpec: "../examples/busybox/Singularity",
 		},
 		{
 			name:       "Debootstrap",
-			dest:       c.env.TestDir + "/container/",
 			dependency: "debootstrap",
 			buildSpec:  "../examples/debian/Singularity",
-			sandbox:    true,
 		},
 		{
 			name:      "DockerURI",
-			dest:      c.env.TestDir + "/container/",
 			buildSpec: "docker://busybox",
-			sandbox:   true,
 		},
 		{
 			name:      "DockerDefFile",
-			dest:      c.env.TestDir + "/container/",
 			buildSpec: "../examples/docker/Singularity",
-			sandbox:   true,
 		},
 		// TODO(mem): reenable this; disabled while shub is down
-		// {"ShubURI", "", "shub://GodloveD/busybox", true},
+		// {
+		// 	name:       "ShubURI",
+		// 	buildSpec:  "shub://GodloveD/busybox",
+		// },
 		// TODO(mem): reenable this; disabled while shub is down
-		// {"ShubDefFile", "", "../examples/shub/Singularity", true},
+		// {
+		// 	name:       "ShubDefFile",
+		// 	buildSpec:  "../examples/shub/Singularity",
+		// },
 		{
 			name:      "LibraryDefFile",
-			dest:      c.env.TestDir + "/container/",
 			buildSpec: "../examples/library/Singularity",
-			sandbox:   true,
 		},
 		{
 			name:      "OrasURI",
-			dest:      c.env.TestDir + "/container/",
 			buildSpec: c.env.OrasTestImage,
-			sandbox:   true,
 		},
 		{
 			name:       "Yum",
-			dest:       c.env.TestDir + "/container/",
 			dependency: "yum",
 			buildSpec:  "../examples/centos/Singularity",
-			sandbox:    true,
 		},
 		{
 			name:       "Zypper",
-			dest:       c.env.TestDir + "/container/",
 			dependency: "zypper",
 			buildSpec:  "../examples/opensuse/Singularity",
-			sandbox:    true,
 		},
 	}
 
 	for _, tc := range tt {
-		// conditionally build a sandbox
-		var args []string
-		if tc.sandbox {
-			args = []string{"--sandbox"}
-		}
-		args = append(args, tc.dest, tc.buildSpec)
+		imagePath := path.Join(c.env.TestDir, "container")
```

Comment:
I was intentionally using `dest:  c.env.TestDir + "/container/"` so that there is a trailing slash at the end of sandbox destination to test against #4407. 
`path.Join` will cleanup things.

Reply:
After this addressed please re-ping me and I will review. Thanks!

Reply:
yeah, sorry about not mentioning this more explicitly.

I have a different commit / PR for that. I think I should be able to upload that one without conflicting with this one...

Reply:
It's #4454

Reply:
that got merged